runtime-modules/constitution/src/lib.rs

```diff
@@ -8,18 +8,34 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 #[cfg(test)]
-mod tests;
+pub(crate) mod tests;
+
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
We don't need to use restrict this to `runtime-benchmarks` since `benchmarking.rs` already has ` #![cfg(feature = "runtime-benchmarks")]`

Reply:
Ok.